src/cordial_sound/scripts/wav_file_publisher.py

```diff
@@ -18,35 +21,66 @@ def __init__(self):
             'cordial/sound/wav/header_length',
             24
         )
-        rospy.Subscriber(rospy.get_param('cordial_sound/play_wav_topic'), String, self.play_wav_file)
-        self._sound_publisher = rospy.Publisher(rospy.get_param('cordial_sound/play_stream_topic'), Sound, queue_size=1)
+        rospy.Subscriber(rospy.get_param(
+            'cordial_sound/play_wav_topic'), String, self.play_wav_file)
+        self._sound_publisher = rospy.Publisher(rospy.get_param(
+            'cordial_sound/play_stream_topic'), Sound, queue_size=10)
         self._pyaudio = pyaudio.PyAudio()
 
+        self.audio_chunks = []
+
+    def single_split_sec(self, from_sec, to_sec, audio):
+        t1 = from_sec * 1 * 1000
```

Comment:
It would be nice to use a constant here instead of hardcoding the `1000`.

It would be nice to set the duration of the sent audio snippet with a ROS parameter, rather than hardcoding it to one second. I see you partially have this functionality in `multiple_split_sec`.